types/stoppable/stoppable-tests.ts

```diff
@@ -1,8 +1,11 @@
 import * as http from 'http';
 import stoppable = require('stoppable');
 
-const server: http.Server = stoppable(http.createServer());
-server.close();
+const server = stoppable(http.createServer());
+server.stop();
```

Comment:
nit: Might be nice to call a function on http.Server so that it shows it is properly extended..

Reply:
Good suggestion, changes have been made.